packages/MarkdownEditor-Core.package/MarkdownInlineParser.class/instance/cutOpener.to..st

```diff
@@ -0,0 +1,10 @@
+parsing
+cutOpener: openingMarkdownDelimiterEmphasis to: aLength
+
+	openingMarkdownDelimiterEmphasis length > aLength 
+		ifTrue: [self tokens 
```

Comment:
in cutCloser: to: you return this whole statement, I'm guessing it should be similar in both methods. Although you seem not to need the return value from this, maybe its better to still return it for consistency